packages/ui/src/components/AccountDetails.tsx

```diff
@@ -66,6 +67,75 @@ export const AccountDetails: FC<Props> = ({ defaultName, headerText, onBack, onC
     Object.values(getValues()).filter((val) => val === '').length > 0 ||
     formState.isSubmitting;
 
+  const renderNewAccountForm = () => (
+    <>
+      <Box mt='m'>
+        <Text color='gray.1'
+          variant='b2m'>
+          Account name
+        </Text>
+      </Box>
+      <TextInput inputRef={register({ required: true })}
+        name='accountName'
+        placeholder='Enter account name' />
+      {errors?.accountName &&
+        <Box>
+          <Text color='alert'
+            variant='b3'>
+            {(errors.accountName).type === 'required' && 'Required field'}
+          </Text>
+        </Box>
+      }
+      <Box mt='m'>
+        <Text color='gray.2'
+          variant='b2'>Please enter a new wallet password below to complete account creation.</Text>
+      </Box>
+      <Password label='Password'
+        withConfirm />
+    </>
+  );
+
+  const renderAccountForm = () => (
+    <>
+      <Box mt='m'>
+        <Text color='gray.2'
+          variant='b2m'>
+          Current wallet account
+        </Text>
+      </Box>
+      <Text color='gray.1'
+        variant='b1'>
+        {selectedAccount && toShortAddress(selectedAccount, { size: 33 })}
+      </Text>
+      <Password label='Wallet password'
+        placeholder='Enter your current wallet password' />
+      <Box my='m'>
+        <Hr />
+      </Box>
+      <Text color='gray.2'
+        variant='b2'>
+        Please enter a new account name to complete restore with your recovery phrase.
```

Comment:
User will go through this form any time they 1) import json, 2) import seed, 3) create a new account. In case of #3, the message above will be confusing.

Reply:
Also, I think it would sound better if we say "...to complete account restora**tion** with your...". I know we're following designs but we could tweak copy to make more sense.

Reply:
> User will go through this form any time they 1) import json, 2) import seed, 3) create a new account. In case of #3, the message above will be confusing.

@remon-nashid that's true; do you think I should remove the message entirely for creating new accounts? Or use a different copy entirely?

e.g.
<img src="https://user-images.githubusercontent.com/7417976/109176300-ab4f6f00-7754-11eb-965c-9e4f7020d0af.png" width="328px" />

Reply:
Let's change it to "....to complete account **addition**" if it sounds good.